content/management/v1/core-resources/collaborators/the-collaborator-object.md

```diff
@@ -0,0 +1,61 @@
+---
+title: The Collaborator Object
+---
+
+This is an object representing collaborators in a space. Some of the properties can be read only and others can be managed by the space owner using the API. You can define the properties of a collaborator such as roles and permissions. 
+
+| Property            | Description          |
+|---------------------|----------------------|
+| `id`                  | Numeric id | 
```

Comment:
I would review the parameters also based on the observations for the add collaborator endpoint

Reply:
These parameters are the object from adding a collaborator with a single role.